README.md

```diff
@@ -136,6 +136,36 @@ sendDistributionMetric(
 
 If your Lambda function is associated with a VPC, you need to ensure it has access to the [public internet](https://aws.amazon.com/premiumsupport/knowledge-center/internet-access-lambda-function/).
 
```

Comment:
I think this title can be misleading, because this only applies when you are not using datadog tracer.

Reply:
I think we should have a section `Trace & Log Correlation` and break it down by using datadog tracer and using xray tracer.

Reply:
Let's move the description for `DD_LOGS_INJECTION` to this new section, since it's getting a little bit complicated.

Reply:
Using the datadog tracer, trace-log correlation should just work, right? Or is there something special they need to do?

Reply:
Correct, it just works if I'm using datadog tracer. However, reading this section as a user, I wouldn't know that. Even if I'm using the datadog tracer, I still might think I need to follow the instructions in this section.

Reply:
I don't think this is a super big deal though, definitely not a blocker for merging this PR, we can always improve the structure later.

Reply:
👍 Updated!